### PR TITLE
Document at least one way to use this repo

### PR DESCRIPTION
**Pull request by lwindolf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,14 @@ This repository is a community-owned library of policies for the OPA Gatekeeper
 
 ## How to use the library
 
-TODO.
+The easiest way to apply a policy from this library is to download and apply the `constrain.yaml` and `template.yaml` provided in each directory
```
Comment (sozercan) [inline README.md:7]:
```suggestion
The easiest way to apply a policy from this library is to download and apply the `constraint.yaml` and `template.yaml` provided in each directory
```

**Review by brycecr — APPROVED** [review]

**Comment by brycecr** [comment]
Needs sign-off but otherwise looks like an improvement. Thanks!

**Comment by lwindolf** [comment]
Sign-off is now done.

**Review by sozercan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -4,8 +4,14 @@ This repository is a community-owned library of policies for the OPA Gatekeeper
...
 ## How to use the library
 
-TODO.
+The easiest way to apply a policy from this library is to download and apply the `constraint.yaml` and `template.yaml` provided in each directory
+
+For example
+
+    cd library/general/httpsonly/
+    kubectl apply -f constraint.yaml
+    kubectl apply -f template.yaml
```
Comment (ritazh) [inline README.md:13]: The deployment order should be template.yaml then constraint.yaml.

**Pull request merged**